src/vsr/replica.zig

```diff
@@ -10885,6 +10978,28 @@ pub fn ReplicaType(
                 (tables_index_block_count + tables_value_block_count +
                     self.state_machine.forest.manifest_log.log_block_checksums.count));
         }
+
+        pub fn log_prefix(self: *const Replica) [2]u8 {
+            var log_message: [2]u8 = undefined;
+
+            log_message[0] = switch (self.replica) {
+                0...9 => 48 + self.replica,
+                else => unreachable,
```

Comment:
Ah wait, I think this is not unreachable -- standbys mean the max replica index is actually 12.